wpiutil/src/main/java/edu/wpi/first/util/sendable2/SendableSet.java

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package edu.wpi.first.util.sendable2;
+
+import java.util.Map;
+import java.util.Set;
+import java.util.WeakHashMap;
+
+public class SendableSet implements AutoCloseable {
+  private Map<SendableTable, Boolean> m_sendables = new WeakHashMap<>();
```

Comment:
What's the intent with the `Boolean` here? A WeakHashSet should be possible, possibly by using `setFromMap`.

Reply:
It looks like Collections.newSetFromMap can be used here, yeah.  It’s effectively equivalent.